packages/@icon-magic/generate/src/plugins/svg-generate.ts

```diff
@@ -80,8 +81,9 @@ export const svgGenerate: GeneratePlugin = {
     // get the mapping object from the metadata for use in dataSupportedDps and useMapForWidthHeight.
     const nameSizeMapping = icon.metadata && icon.metadata.nameSizeMapping;
 
-    // assign value of param or else set to false if undefined. This will be used to add or replace width and height.
-    const useMapForWidthHeight = params.useNameSizeMapForWidthHeight ?? false;
+    // will be used by svgo to remove width and height. If needed, will add width and height from nameSizeMapping.
+    // turn into a boolean in case it was undefined.
+    let isFixedDimensions = !!params.isFixedDimensions;
```

Comment:
Suggest calling this variable something else like, shouldAddWidthHeight so that you don't have to set it to false again later.